lib/livebook_web/live/session_live/secrets_component.ex

```diff
@@ -49,22 +50,34 @@ defmodule LivebookWeb.SessionLive.SecretsComponent do
                 <%= for {secret_name, _} <- Enum.sort(@secrets) do %>
                   <.secret_with_badge
                     secret_name={secret_name}
+                    origin="session"
                     stored="Session"
                     action="select_secret"
                     active={secret_name == @prefill_secret_name}
                     target={@myself}
                   />
                 <% end %>
-                <%= for {secret_name, _} <- livebook_only_secrets(@secrets, @livebook_secrets) do %>
+                <%= for %{name: secret_name, origin: :app} <- @livebook_secrets do %>
```

Comment:
Shouldn't we sort here? Or maybe we should sort when we get them?

Reply:
We don't need to sort, since we're already sorting from `Secrets.get_secrets/0`